functionaltests/main_test.go

```diff
@@ -183,77 +182,12 @@ func allDataStreams(namespace string) []string {
 	return res
 }
 
-// getDocsCountPerDS retrieves document count.
-func getDocsCountPerDS(t *testing.T, ctx context.Context, esc *esclient.Client) (esclient.DataStreamsDocCount, error) {
+// getDocCountPerDS retrieves document count per data stream.
+func getDocCountPerDS(t *testing.T, ctx context.Context, esc *esclient.Client) (esclient.DataStreamsDocCount, error) {
```

Comment:
We should probably remove this helper function, what do you think? Does it add value?

Reply:
Great idea moving the `NoError` check in this function 🚀